src/rev_ai/models/error.py

```diff
@@ -0,0 +1,112 @@
+class ProblemDetail:
+	def __init__(
+		self,
+		status,
+		type_,
+		title,
+		detail
+	):
+		self.status = status
+		self.type = type_
+		self.title = title
+		self.detail = detail
+
+class ApiError:
```

Comment:
https://docs.python.org/2/tutorial/errors.html#tut-userexceptions

Reply:
the only related difference in python3 is that there is some (here irrelevant) syntactic sugar for certain class relationships